pkg/ngprofile/detach.go

```diff
@@ -0,0 +1,41 @@
+package ngprofile
+
+import (
+	"context"
+	"fmt"
+	argoclient "github.com/argoproj/argo-cd/v2/pkg/apiclient"
+	clusterpkg "github.com/argoproj/argo-cd/v2/pkg/apiclient/cluster"
+	"github.com/arlonproj/arlon/pkg/app"
+)
+
+func DetachFromCluster(argoIf argoclient.Client, profName string, clusterName string,
+) (modified bool, err error) {
+	conn, clustIf, err := argoIf.NewClusterClient()
+	if err != nil {
+		err = fmt.Errorf("failed to get argocd cluster client: %s", err)
+		return
+	}
+	defer conn.Close()
+	clust, err := clustIf.Get(context.Background(), &clusterpkg.ClusterQuery{Name: clusterName})
+	if err != nil {
+		err = fmt.Errorf("failed to get argocd cluster: %s", err)
+		return
+	}
+	if clust.Labels == nil {
+		clust.Labels = make(map[string]string)
+	}
```

Comment:
If the `Lables == nil`, then can we conclude that the cluster doesn't have that profile added to it instead of initialising the Labels?